### PR TITLE
Adding stats for upgrade test and updating to aetos

### PR DESCRIPTION
**Pull request by lsrinivas-pure**
**What this PR does / why we need it**:
Created a stats struct and push data to aetos for PX upgrade

**Which issue(s) this PR fixes** (optional)
Closes #PTX-14872

**Special notes for your reviewer**:
http://aetos.pwx.purestorage.com/resultSet/testSetID/67685 

sample result from aetos

```
{
    "_id": {
      "$oid": "63c050a31c575913d7ce0846"
    },
    "name": "px-upgrade-stats",
    "product": "px-enterprise",
    "statsType": "upgrade",
    "version": "0cc6b62",
    "data": {
      "dash-url": "http://aetos.pwx.purestorage.com/resultSet/testSetID/67685",
      "duration": "2 mins",
      "fromVersion": "2.12.2-0cc6b62",
      "status": "PASS",
      "toVersion": "2.12.2-0cc6b62"
    },
    "dateTime": "2023-01-12 18:25:39+00:00"
  }
```

### CODE REVIEW

File: pkg/aetosutil/dashboardutil.go [hunk 1]
```diff
@@ -513,3 +521,26 @@ func Get() *Dashboard {
 	}
 	return dash
 }
+
+func (d *Dashboard) UpdateStats(name, product, statType, version string, dashStats map[string]string) {
+	if d.IsEnabled {
+
+		dashStats["dash-url"] = fmt.Sprintf("http://aetos.pwx.purestorage.com/resultSet/testSetID/%d", d.TestSetID)
```
Comment (CharudathGopal) [inline pkg/aetosutil/dashboardutil.go:528]: There is a constant defined for AETOS URL, can you please use that here and in other places

Comment (lsrinivas-pure) [reply pkg/aetosutil/dashboardutil.go]: fixed

**Review by CharudathGopal — APPROVED** [review]

**Review by nikolaypopov — DISMISSED** [review]
Can we hold off on this PR please, I am about to merge the PR where I re-wrote these tests

**Comment by nikolaypopov** [comment]
I am done with my changes, please resolve conflicts and then it should be good

**Pull request merged**